controllers/graph.controller.js

```diff
@@ -0,0 +1,84 @@
+const graphDbService = require("../services/graph.service");
+const {
+  ERROR_GRAPH_NOT_FOUND,
+  ERROR_INVALID_GRAPH,
+  INTERNAL_ERROR,
+} = require("../constants");
+const { isValidGraph } = require("../services/utils");
+
+const publishGraph = async (req, res) => {
+  try {
+    const { adjacencyLists, userId } = req.body;
+    if (!isValidGraph(adjacencyLists)) {
+      res.send({ error: ERROR_INVALID_GRAPH });
+      return;
+    }
+    const newGraph = await graphDbService.DbCreateGraph({
+      adjacencyLists,
+      userId,
+    });
+    return res.send(newGraph);
+  } catch (err) {
+    console.log(err);
+    res.send({ error: INTERNAL_ERROR });
```

Comment:
PLease remove the console, we do not want to have consoles in the production code. 
Good job on sending the error to the user